src/js/personalization/va-profile-beta/utils/tests/address.unit.spec.js

```diff
@@ -0,0 +1,118 @@
+import * as addressUtils from '../address';
+import { expect } from 'chai';
+
+const domestic = {
+  type: 'DOMESTIC',
+  countryName: 'USA',
+  addressOne: 'First',
+  addressTwo: 'Second',
+  addressThree: 'Third',
+  city: 'City',
+  stateCode: 'KY',
+  zipCode: '12345'
+};
+
+const international = {
+  type: 'INTERNATIONAL',
+  countryName: 'Canada',
+  addressOne: 'First',
+  addressTwo: 'Second',
+  addressThree: 'Third',
+  city: 'Montreal'
+};
+
+const military = {
+  type: 'MILITARY',
+  addressOne: 'First',
+  addressTwo: 'Second',
+  addressThree: 'Third',
+  militaryPostOfficeTypeCode: 'APO',
+  militaryStateCode: 'AA',
+  zipCode: '12345'
+};
+
+describe('formatAddress', () => {
+
+  it('formats domestic addresses', () => {
+    const expectedResult = {
+      street: 'First, Second Third',
+      cityStateZip: 'City, Kentucky 12345',
+      country: ''
+    };
+
+    expect(addressUtils.formatAddress(domestic)).to.deep.equal(expectedResult);
+  });
+
+  it('formats military addresses', () => {
+    const expectedResult = {
+      street: 'First, Second Third',
+      cityStateZip: 'APO, AA 12345',
+      country: ''
+    };
+
+    expect(addressUtils.formatAddress(military)).to.deep.equal(expectedResult);
+  });
+
+  it('formats international addresses', () => {
+    const expectedResult = {
+      street: 'First, Second Third',
+      cityStateZip: 'Montreal',
+      country: 'Canada'
+    };
+
+    expect(addressUtils.formatAddress(international)).to.deep.equal(expectedResult);
+  });
+
+});
+
+describe('getStateName', () => {
+
+  it('gets the full name of a state from its abbreviation', () => {
+    expect(addressUtils.getStateName('KY')).to.equal('Kentucky');
+    expect(addressUtils.getStateName('ky')).to.equal('Kentucky');
+  });
+
+});
+
+describe('isEmptyAddress', () => {
```

Comment:
isIncomplete might be more descriptive?

Reply:
I used some of the naming conventions from Letters to decide that name but I agree that is better. I'll update that.

Reply:
Ah actually on second thought, there can be incomplete addresses that still contain street information or other properties, but aren't considered empty